common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -153,19 +157,18 @@ export class SendTransaction extends React.Component {
     if (this.state.generateDisabled !== !this.isValid()) {
       this.setState({ generateDisabled: !this.isValid() });
     }
-
     const componentStateTransaction = this.state.transaction;
     if (componentStateTransaction) {
       // lives in redux state
-      const currentTxAsBroadcastTransaction = getTxFromBroadcastStatusTransactions(
+      const currentTxAsSignedTransaction = getTxFromBroadcastTransactionStatus(
         this.props.transactions,
         componentStateTransaction.signedTx
       );
       // if there is a matching tx in redux state
-      if (currentTxAsBroadcastTransaction) {
+      if (currentTxAsSignedTransaction) {
         // if the broad-casted transaction attempt is successful, clear the form
-        if (currentTxAsBroadcastTransaction.successfullyBroadcast) {
-          this.resetTransaction();
+        if (currentTxAsSignedTransaction.successfullyBroadcast) {
```

Comment:
Can combine these into a single if- statement to avoid nesting

Reply:
I'm probably missing something, but I don't think I can in this case, as checking for truthiness of `componentStateTransaction` is necessary before `currentTxAsSignedTransaction` can be initialized.